extensions/apiconnect/src/__tests__/acceptance/apiconnect.acceptance.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/apiconnect
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application} from '@loopback/core';
+import {RestApplication, RestServer} from '@loopback/rest';
+import {expect} from '@loopback/testlab';
+import {ApiConnectBindings, ApiConnectComponent} from '../..';
+import {ApiConnectSpecOptions} from '../../apiconnect.spec-enhancer';
+
+describe('LoopBack Extension for IBM API Connect', () => {
```

Comment:
nitpick: making it more descriptive, I'm thinking whether it's better if the description is:
`Apply IBM API Connect Extension to LoopBack application`